src/Owin.Security.Providers.SlackProvider/Provider/SlackAuthenticatedContext_Webhook.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Newtonsoft.Json.Linq;
+
+namespace Owin.Security.Providers.Slack.Provider
+{
+    public class SlackAuthenticatedContext_Webhook
+    {
+        public SlackAuthenticatedContext_Webhook()
+        {
+
+        }
+        public SlackAuthenticatedContext_Webhook(JObject botnode)
```

Comment:
Can you please have this file accept just the url, channel, and configuration_url in the constructor, and not have it also be responsible for parsing the jobject?